### ISSUE TITLE
Rewind and fast forward on Samsung tv app

### ISSUE THREAD

**Issue by Benmyk** [Stremio/stremio-features#645]
It appears seeking through a movie or show is very difficult as the fast forward or rewind feature in Samsung tv isn't available, if this can be fixed it will be appreciated . thanks

**Comment by dr0vur** [comment]
+1 for this. It's basically there, just not visible

**Comment by Benmyk** [comment]
> +1 for this. It's basically there, just not visible. If you highlight pause and then move down one and use the left or right you can skip forward and backward. It just needs to be adjusted in visibility and show the progress of the video also. The android tv app has this, just not the samsung one.

It used to, but stopped working , cuz I remember when after the update when I highlight the pause and bring it down one it sends me straight to subtitles , and the seeking is soo slow to be honest

**Comment by dr0vur** [comment]
Mine still works tbh. Although alow yes, i just move one down and press right 5-6 times to skip intro :) Would be a pain to try seek to a specific time

**Comment by Benmyk** [comment]
> Mine still works tbh. Although alow yes, i just move one down and press right 5-6 times to skip intro :) Would be a pain to try seek to a specific time 

Yeah hope it gets fixed, appears the video playback and display has been improved in the latest update

**Comment by Troughy** [comment]
sadly I still can't fast forward on Samsung TV

**Comment by kKaskak** [comment]
hi this was implemented and works as expected on samsung tvs
